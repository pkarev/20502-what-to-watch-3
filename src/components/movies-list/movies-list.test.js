import React from 'react';
import renderer from 'react-test-renderer';
import MoviesList from './movies-list';

const movies = [
  {
    name: `Fantastic Beasts`,
    poster: `/img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    name: `Bohemian Rhapsody`,
    poster: `/img/bohemian-rhapsody.jpg`,
  },
  {
    name: `Moonrise kindom`,
    poster: `/img/moonrise-kingdom.jpg`
  }
];

it(`Render MoviesList`, () => {
  const tree = renderer
    .create(<MoviesList movies={movies}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
