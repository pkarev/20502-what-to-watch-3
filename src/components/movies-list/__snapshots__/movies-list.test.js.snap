// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MoviesList 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        controls={false}
        muted={true}
        poster="/img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        style={
          Object {
            "height": "100%",
            "objectFit": "cover",
            "objectPosition": "center center",
            "width": "100%",
          }
        }
      >
        <source
          src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/1"
        onClick={[Function]}
      >
        Fantastic Beasts
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        controls={false}
        muted={true}
        poster="/img/bohemian-rhapsody.jpg"
        style={
          Object {
            "height": "100%",
            "objectFit": "cover",
            "objectPosition": "center center",
            "width": "100%",
          }
        }
      >
        <source
          src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/2"
        onClick={[Function]}
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        controls={false}
        muted={true}
        poster="/img/moonrise-kingdom.jpg"
        style={
          Object {
            "height": "100%",
            "objectFit": "cover",
            "objectPosition": "center center",
            "width": "100%",
          }
        }
      >
        <source
          src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
        />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/3"
        onClick={[Function]}
      >
        Moonrise kindom
      </a>
    </h3>
  </article>
</div>
`;
